src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1921,6 +1921,7 @@ internal enum ErrorCode
         #endregion diagnostics introduced for C# 9.0
 
         ERR_UnexpectedVarianceStaticMember = 9100,
+        ERR_EqualityContractRequiresGetter = 9101,
```

Comment:
>ERR_EqualityContractRequiresGetter = 9101, [](start = 8, length = 42)

It looks like this branch is significantly behind. I think in master the last used error number is 8905